src/main/java/mat/DTO/fhirconversion/CqlConversionResult.java

```diff
@@ -0,0 +1,30 @@
+package mat.DTO.fhirconversion;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Transient;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+public class CqlConversionResult {
+    ConversionType type;
+    Boolean result;
+    List<String> errors = new ArrayList();
+    @Transient
```

Comment:
I would remove these @Transient annotations. 
I don't even think it is even the right annotation to use here on the ms side they should have used the jackson one. I know the fhirCql still comes across.even with it sent as @Transient.

Reply:
Sure, How about access modifiers for other few variables?

Reply:
ah Codacy ... I just make them all private when I see that

Reply:
ok, will make them private